wsmaster/che-core-api-devfile/README.md

```diff
@@ -0,0 +1,187 @@
+
+
+### Introduction
+Previously, the  two kind of recipes was available to bootstrap a cloud developer workspace and make it portable: [Chefile](https://www.eclipse.org/che/docs/chefile.html) 
+and [Factories](https://www.eclipse.org/che/docs/factories-getting-started.html#try-a-factory).
+As a continuation of this, the brand new `devfile` format was introduced, which unites the forming simplicity and the high variety of different tools available to develop a container based application.
+
+### What the devfile consists of
+The minimal devfile sufficient to run a workspace from it, consists of the following parts:
+ - Specification version
+ - Name
+ - Tools set 
+ 
+To get more functional workspace, the following parts can be added:
+ - Project(s)
+ - Command(s)
```

Comment:
s/Command(s)/A list of commands: actions to manage the workspace components like running the dev tools, starting the runtime environments etc.../